ttnn/cpp/pybind11/operations/binary.hpp

```diff
@@ -135,32 +144,35 @@ Example::
     detail::bind_binary(
         module,
         ttnn::subtract_,
-        R"doc(subtract_(input_tensor_a: ttnn.Tensor, input_tensor_b: Union[ttnn.Tensor, int, float], *, memory_config: Optional[ttnn.MemoryConfig] = None, dtype: Optional[ttnn.DataType] = None) -> ttnn.Tensor
+        R"doc(subtract_(input_tensor_a: ttnn.Tensor, input_tensor_b: Union[ttnn.Tensor, int, float], *, memory_config: Optional[ttnn.MemoryConfig] = None, dtype: Optional[ttnn.DataType] = None, activations: Optional[List[str]] = None) -> ttnn.Tensor
 
-    Subtracts :attr:`input_tensor_b` from :attr:`input_tensor_a` and returns the tensor with the same layout as :attr:`input_tensor_a` in-place
+        Subtracts :attr:`input_tensor_b` from :attr:`input_tensor_a` and returns the tensor with the same layout as :attr:`input_tensor_a` in-place
```

Comment:
Not familiar with binding docs from C++. Does this extra tab makes doc shifted right too? Is it intentional?

Reply:
@ayerofieiev-tt I just wanted to indent the whole file the same way. It should affect how the documentation
looks like

Reply:
@arakhmati it makes c++ file look better, but does it affect docs? If yes, are we good with that?